pkg/proof/proof.go

```diff
@@ -33,6 +34,12 @@ type FullProof struct {
 // GenerateZkProof executes snarkjs groth16prove function and returns proof only if it's valid
 func GenerateZkProof(ctx context.Context, circuitPath string, inputs ZKInputs) (*FullProof, error) {
 
+	config, err := configs.ReadConfigFromFile("prover")
```

Comment:
Needed config options should be provided from outside of this function.

Reply:
Done